C/casacore/build_tarballs.jl

```diff
@@ -104,14 +107,15 @@ products = Product[
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="FFTW_jll", uuid="f5851436-0d7a-5f13-b9de-f02708fd171a"); compat = "3.3.10")
-    Dependency(PackageSpec(name="CFITSIO_jll", uuid="b3e40c51-02ae-5482-8a39-3ace5868dcf4"); compat = "4.3.1")
-    Dependency(PackageSpec(name="WCS_jll", uuid="550c8279-ae0e-5d1b-948f-937f2608a23e"); compat = "7.7.0")
-    Dependency(PackageSpec(name="Readline_jll", uuid="05236dd9-4125-5232-aa7c-9ec0c9b2c25a"); compat = "8.1.1")
-    Dependency(PackageSpec(name="GSL_jll", uuid="1b77fbbe-d8ee-58f0-85f9-836ddc23a7a4"); compat = "2.7.2")
-    Dependency(PackageSpec(name="OpenBLAS32_jll", uuid="656ef2d0-ae68-5445-9ca0-591084a874a2"))
-    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
+    Dependency(PackageSpec(name="FFTW_jll", uuid="f5851436-0d7a-5f13-b9de-f02708fd171a"); compat="3.3.10");
+    Dependency(PackageSpec(name="CFITSIO_jll", uuid="b3e40c51-02ae-5482-8a39-3ace5868dcf4"); compat="4.4.0");
+    Dependency(PackageSpec(name="WCS_jll", uuid="550c8279-ae0e-5d1b-948f-937f2608a23e"); compat="7.7.0");
+    Dependency(PackageSpec(name="Readline_jll", uuid="05236dd9-4125-5232-aa7c-9ec0c9b2c25a"); compat="8.1.1");
+    Dependency(PackageSpec(name="GSL_jll", uuid="1b77fbbe-d8ee-58f0-85f9-836ddc23a7a4"); compat="2.7.2");
+    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"); compat="5.11.2");
```

Comment:
LBT v5.4 should be enough
```suggestion
    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"); compat="5.4");
```
Note that if you're using the LP64 API you need to make sure that your consumer package (whatever is using this package) provides a backing LP64 BLAS library: by default Julia only provides an ILP64 one.

Reply:
What are the implications of this? As in, the likes of [Casacore.jl](https://github.com/torrance/Casacore.jl) should add a dep on OpenBLAS?

Reply:
More OpenBLAS32

Reply:
So would it be better to just leave this as a dep on OpenBLAS32?

Reply:
Maybe yes (but make sure the library itself links to lbt, rather than openblas).

Reply:
I'm not quite sure I follow. Are you saying to have both a dep on OpenBLAS32 and LBT but tell cmake to use lbt as the blas/lapack provider?

Reply:
> Are you saying to have both a dep on OpenBLAS32 and LBT

Yes.

> but tell cmake to use lbt as the blas/lapack provider?

Yes, which is what you're already doing, but double check that cmake actually does it.